spec/models/video_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe Video do
+  it { should belong_to(:category) }
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:description) }
+
+  describe :search_by_title do 
+    before(:each) do 
+      @video_one = Video.create(title: 'Video one', description: 'Video one is awesome', created_at: 1.day.ago)
```

Comment:
Using instance variables in RSpec is no. Use let or let! instead to either lazy load or create.